docs/source/jobs.rst

```diff
@@ -39,7 +39,7 @@ Jobs in the GFS Configuration
 | analdiag          | Creates netCDF diagnostic files containing observation values, innovation (O-F), error, quality control, as well as   |
 |                   | other analysis-related quantities (cnvstat, radstat, ozstat files).                                                   |
 +-------------------+-----------------------------------------------------------------------------------------------------------------------+
-| arch              | Archives select files from the deterministic model and cleans up older data.                                          |
+|arch_vrfy/arch_tars| Archives select files from the deterministic model and cleans up older data.                                          |
```

Comment:
Suggest splitting these up.  They are separate tasks.  `vrfy` archives the files locally that are needed for verification.  `tars` is an optional job that backs up the COM data structure.